packages/react-server-cli/src/compileClient.js

```diff
@@ -250,7 +250,11 @@ module.exports = {
 							});`);
 			} else {
 				routesOutput.push(`
-							cb(unwrapEs6Module(require("${relativePathToPage}")));`);
+							try {
+								cb(unwrapEs6Module(require("${relativePathToPage}")));
+							} catch (e) {
+								console.error('Failed to load page at "${relativePathToPage}"', e.stack)
+							}`);
```

Comment:
This is part of #80 that I re-did in place because I needed to unmask some errors during development.  Forgot to undo before committing, and now it's buried in history.  I can squash if it bugs you, but I won't unless prompted.